lib/Optimizer/Lower.cpp

```diff
@@ -794,6 +794,18 @@ static void lowerBatchMatMulNode(Function *F, CompilationContext &cctx,
   replaceAllUsesOfWith(cctx.loweredInfoMap, BMMN.getResult(), RN);
 }
 
+static void lowerSparseLengthsSumNode(Function *F, CompilationContext &cctx,
```

Comment:
what's the reason for this given you added implementation of SLS to interpreter and CPU backend?

Reply:
It is meant for other backends that does not have SLS implementation, i.e. OpenCL.
I should have thought about it before implementing for interpreter and CPU.